applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_fast_pair.conf

```diff
@@ -97,6 +106,18 @@ CONFIG_PWM=y
 CONFIG_LED=y
 CONFIG_LED_PWM=y
 
+# Enable MCUmgr Bluetooth transport and increase Bluetooth buffers to speed up DFU image transfer.
+CONFIG_MCUMGR_TRANSPORT_BT=y
+CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=n
+
+CONFIG_IMG_MANAGER=y
+CONFIG_MCUBOOT_IMG_MANAGER=y
```

Comment:
The CONFIG_IMG_MANAGER=y is also needed for config channel DFU module which does not select it automatically. If we remove it, disabling CONFIG_DESKTOP_DFU_MCUMGR_ENABLE  would lead to build failures.

The linked case uses only DFU over `CONFIG_DESKTOP_DFU_MCUMGR_ENABLE` so that there is no need to set the option explicitly there